test/backend-cpp/TestCategoryMapper.cpp

```diff
@@ -23,8 +23,8 @@ static bool testInt64ToStr() {
 
   const CategoryMapperBuilder::CMAttributes attributes = {{1, 2, 3, 4, 5},
       {"cat", "dog", "human", "tiger", "beaver"}, -1, "unknown"};
-  const llvm::ArrayRef<int64_t> input = {1, 2, 3, 6, 4, 5};
-  const llvm::ArrayRef<const char *> expResult = {
+  const llvm::SmallVector<int64_t> input = {1, 2, 3, 6, 4, 5};
```

Comment:
It is more efficient to use a concrete value, say `llvm::SmallVector<int64_t, 6>`. Same for other places.

Reply:
@tungld Fixed. Thank you!